webvtt.go

```diff
@@ -70,18 +72,21 @@ func parseTimestampMapWebVTT(line string) (timeOffset time.Duration, err error)
 			local, err = parseDurationWebVTT(splits[1])
 			if err != nil {
 				err = fmt.Errorf("astisub: parsing webvtt duration failed: %w", err)
+				timestampMap = nil
 				return
 			}
+			timestampMap.LocalTimestamp = local
 		case "mpegts":
 			mpegts, err = strconv.ParseInt(splits[1], 10, 0)
 			if err != nil {
 				err = fmt.Errorf("astisub: parsing int %s failed: %w", splits[1], err)
+				timestampMap = nil
 				return
 			}
+			timestampMap.PresentationTimestamp = mpegts
 		}
 	}
 
-	timeOffset = time.Duration(mpegts)*time.Second/90000 - local
```

Comment:
Could you add 
```go
timestampMap = &WebVTTTimestampMap{
  Local: local,
  MpegTS: mpegts,
}
```

Reply:
Can you explain the reasoning behind that? I'd rather return a nil + error at this point of the function since parsing failed and a non nil `&WebVTTTimestampMap` will have default values for int64 and time.Duration for both fields implying it was in some way valid.

Reply:
Oh I see. You're leaning on the naked returns. They can be a bit confusing for me. It's a feature of golang I'm not the biggest fan of because I feel it tends to make the code a bit harder for me to follow